expected_output:
  - Pod logging-agent is experiencing a CrashLoopBackOff
  - The pod attempts to execute a command wge which is not found
  - Mention `wge` might be a typo and the intended command may be `wget`
before_test: |
  kubectl apply -f https://raw.githubusercontent.com/robusta-dev/kubernetes-demos/main/init_crashloop_backoff/create_init_crashloop_backoff.yaml
  sleep 40
after_test: |
  kubectl delete -f https://raw.githubusercontent.com/robusta-dev/kubernetes-demos/main/init_crashloop_backoff/create_init_crashloop_backoff.yaml
evaluation:
  correctness: 1
